packages/flutter_adaptive_scaffold/CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
-## NEXT
+## 0.1.9
+
+* FIX : Drawer stays open even on destination tap - [flutter/flutter#141938](https://github.com/flutter/flutter/issues/141938)
+
+## 0.1.8+1
```

Comment:
I'd just eliminate these lines and put all the changes into 0.1.9, since 0.1.8+1 will not be published.

Reply:
Removed 0.1.8+1 and merged those change sets into 0.1.9